onnxruntime/test/framework/memcpy_transformer_test.cc

```diff
@@ -16,16 +16,16 @@ using namespace ONNX_NAMESPACE;
 namespace onnxruntime {
 namespace test {
 
-typedef std::vector<onnxruntime::NodeArg*> ArgMap;
+typedef std::vector<NodeArg*> ArgMap;
```

Comment:
Most diffs here are from removing the unnecessary `onnxruntime::` prefix.

2 tests are updated to assign the 'If' node in the main graph to an EP. The existing test is unchanged apart from the assignment. Test is moved to a lambda with the 'If' node being assigned to the CPU and CUDA EPs.

- lines 195 and 350 is where the existing test was moved into a lambda
- lines 258 and 400 is where the assignment of the 'If' node occurs